app/utils/IncidentP5TransportEquipmentHelper.scala

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright 2024 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package utils
+
+import generated.TransportEquipmentType07
+import models.Index
+import play.api.Logging
+import play.api.i18n.Messages
+import uk.gov.hmrc.govukfrontend.views.viewmodels.summarylist.SummaryListRow
+import viewModels.sections.Section.{AccordionSection, StaticSection}
+
+class IncidentP5TransportEquipmentHelper(
+  data: Seq[TransportEquipmentType07]
```

Comment:
This can just be an individual `TransportEquipmentType07`, then we don't need to pass an index to all the methods